api/serviceTypes/serviceTypeRouter.js

```diff
@@ -122,10 +122,11 @@ router.get('/:id', (req, res, next) => {
  */
 router.post('/', canCrudServiceType, (req, res, next) => {
   ServiceTypes.createServiceType(req.body)
-    .then((newServiceType) => {
+    .then((newServiceTypeData) => {
       res.status(201).json({
         message: 'New service type created',
-        service_type: newServiceType,
+        service_type: newServiceTypeData.service_type,
+        service_type_programs: newServiceTypeData.service_type_programs,
```

Comment:
I just want to check on naming conventions throughout the backend. Is having "data" at the end of the name common in other parts of the code as well?

Reply:
Don't remember why we made that change. Agree that it would be cleaner without "Data" suffix.